.circleci/config.yml

```diff
@@ -16,19 +16,27 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "package.json" }}
+          - v2-dependencies-{{ checksum "package.json" }}
           # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
+          - v2-dependencies-
 
-      - run: yarn install
+      - run: yarn install --frozen-lockfile
 
       - save_cache:
           paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
+            - ~/.cache
```

Comment:
The [Cypress Docs](https://docs.cypress.io/guides/guides/continuous-integration.html#Caching) state that we should be caching `~/.cache` instead of node_modules to ensure that the right version of Cypress is installed. It also indicates that the JS dependencies should still be cached correctly.

However, I'm not seeing that on our circle builds, and it seems like we're still re-downloading everything now. May need your help with this when you're back, @juanca